release.nix

```diff
@@ -8,7 +8,7 @@
 # Note:
 # Verify that .ci/instantiate-all.nix lists the expected paths when adding to this file.
 let
-  mobileReleaseTools = (import ./lib/release-tools.nix);
+  mobileReleaseTools = (import ./lib/release-tools.nix {});
```

Comment:
Because release-tools.nix now imports `<nixpkgs>` as its default value for the `pkgs` argument, this is equivalent to the previous behavior.